packages/llvm.rb

```diff
@@ -3,78 +3,74 @@
 class Llvm < Package
   description 'The LLVM Project is a collection of modular and reusable compiler and toolchain technologies. The optional packages clang, lld, lldb, polly, compiler-rt, libcxx, libcxxabi, and openmp are included.'
   homepage 'http://llvm.org/'
-  @_ver = '11.1.0-rc1'
+  @_ver = '11.1.0'
   version @_ver
   compatibility 'all'
   source_url "https://github.com/llvm/llvm-project/archive/llvmorg-#{@_ver}.tar.gz"
-  source_sha256 'e610297041129a5c5b24355a988c99c7452ee7105ee2355334a8c521b988eb3c'
+  source_sha256 '53a0719f3f4b0388013cfffd7b10c7d5682eece1929a9553c722348d1f866e79'
 
-  binary_url ({
-     aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/llvm-11.1.0-rc1-chromeos-armv7l.tar.xz',
-      armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/llvm-11.1.0-rc1-chromeos-armv7l.tar.xz',
-        i686: 'https://dl.bintray.com/chromebrew/chromebrew/llvm-11.1.0-rc1-chromeos-i686.tar.xz',
-      x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/llvm-11.1.0-rc1-chromeos-x86_64.tar.xz',
+  binary_url({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/llvm-11.1.0-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/llvm-11.1.0-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/llvm-11.1.0-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/llvm-11.1.0-chromeos-x86_64.tar.xz'
   })
-  binary_sha256 ({
-     aarch64: '5ecb72e60483b15c7588446a7ac424d64cdd3f21ca0fa5ffea4737a25dedca89',
-      armv7l: '5ecb72e60483b15c7588446a7ac424d64cdd3f21ca0fa5ffea4737a25dedca89',
-        i686: '5595ca097aa454bd54b8a336d467ee266f8afd4bd8100d1a3831e3face9b4f43',
-      x86_64: '031f6f3510ebbc5a1270fac07934d7efa9126c8e89f98695505b1376d6d233a5',
+  binary_sha256({
+    aarch64: 'ba0520a8d5bfa90efb3eb13a530c8d4052f9e7572aa30dd42dfc624b88a0251e',
+     armv7l: 'ba0520a8d5bfa90efb3eb13a530c8d4052f9e7572aa30dd42dfc624b88a0251e',
+       i686: '3a514f96c846d777c838fef6c585d2278abf2a586bfaeacfd3e2bccc9ffce838',
+     x86_64: '5b9e01c2411ea174293fc1d33b2dbf91f7001eb3cc3eda2e849b80f6f0dcb3c8'
   })
-  
-  # llvm_stage1 is compiled with gcc, without -flto
-  # Then in stage 2 (llvm.rb) everything is compiled with clang built in stage 1, with -flto=thin
-  # FYI LLVM 11.1.0 is a tiny rebuild of 11.0.1 fixing ABI compatibility with LLVM 10.0.0 & LLVM 12
 
   depends_on 'ocaml' => :build
-  depends_on 'libedit'
-  depends_on 'libtirpc'
-  depends_on 'swig'
   depends_on 'pygments' => :build
   depends_on 'ccache' => :build
-  depends_on 'llvm_stage1' => :build
-  depends_on 'binutils' => :build
+  # depends_on 'llvm_stage1' => :build # Not needed with a functional
+  # prior llvm 11 build.
```

Comment:
Are you sure we need to remove all these dependencies?

Reply:
The llvm_stage1 is commented on purpose.

Binutils is a core package, no?

Reply:
Right but you need to be careful about removing dependencies from core packages because those dependencies actually make up the entire list of core packages. :)  I remember this package was added after binutils so yes, this line can be removed.  Just saying though.  I'm guessing you answered my question with a yes.

Reply:
hmmm... we need to modify a core packages cleaning script which doesn't touch core package files... or actually handle dependencies properly... :)